src/providers/wms/qgswmsprovider.cpp

```diff
@@ -3040,6 +3054,7 @@ QgsRasterIdentifyResult QgsWmsProvider::identify( const QgsPointXY &point, QgsRa
     else if ( !getFeatureInfoUrl().isNull() )
     {
       // KVP
+      QgsDebugMsg( QStringLiteral( "KVP ignore GetFeatureURL %1" ).arg( mSettings.mIgnoreGetFeatureInfoUrl ) );
```

Comment:
If this is an error/warning level condition it's ok to use QgsDebugMsg better use QgsDebugMsgLevel otherwise.